hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherOpProcessor.java

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2018-2019 "Neo4j, Inc." [https://neo4j.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/**
+ * Description of the modifications:
+ * <p>
+ * 1) Changed the method signature to adopt the gremlin-server 3.5.1.
+ * public Optional<ThrowingConsumer<Context>> selectOther(RequestMessage requestMessage)
+ * -->
+ * public Optional<ThrowingConsumer<Context>> selectOther(Context ctx)
+ * <p>
+ * 2) Changed the package name.
+ * org.opencypher.gremlin.server.op.cypher
```

Comment:
can we just import it, or move this reference files to an independent package like `org.apache.hugegraph.api.cypher.opencypher`?

Reply:
I've made some chanages to adapt to the gremlin-3.5.1 version, as the original one is incompatible.
org.apache.hugegraph.opencypher, is that ok?

Reply:
I think it's ok